src/components/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "components",
```

Comment:
Why did you chose this approach? We need to create package.json file and also disable eslint for imports. Could you compare first and third approaches https://www.novis.co/post/custom-aliases-in-react-native-with-babel/?

Reply:
I have chooseen this approach because it was the simplest way and it does not requires additional packages.

Reply:
Each person should disable eslint for each import and package.json files looks strange in each folder. I don't think so that it more simplest way. Let's try different approach and compare.

Reply:
Implementation with aliases of .babelrc did not work in React Native. Investigation of the issue lead me to https://github.com/tleunen/babel-plugin-module-resolver/issues/310 
I propose to leave current solution with package.json